.github/workflows/ci.yml

```diff
@@ -9,7 +9,7 @@ on:
       - 'trying'
 
 jobs:
-  markdownlint:
+  super-linter:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
```

Comment:
`- name: Run Markdown Lint` should be `- name: Run Super Linter` (I'd put suggestion directly on it, but it's out of purview for the review changes part.

Reply:
Thanks! Fixed in 37c47d7d27899fcacc2cdff10b6adfae55b89c26.